manage.py

```diff
@@ -199,9 +199,8 @@ def main():
         "start", help="Start a local version of a model inference"
     )
     parser_start.add_argument(
-        "--model-id",
+        "model",
         type=str,
-        required=True,
```

Comment:
shouldn't this be required?

Reply:
No, positional arguments are necessarily required (it actually raises an exception if that flag is on :)